code/api/views.py

```diff
@@ -425,14 +425,35 @@ def get(self, request, author_id, post_id):
 
 
         try:
-            post = LocalPost.objects.get(id=post_id)
-            authors = [like.author.as_json() for like in post.likes.all()]
+            author = get_object_or_404(Author, id=author_id)
+            post = get_object_or_404(LocalPost, id=post_id, author=author)
+
+            post_likes = post.likes.all()
+
+            items = []
+            for like in post_likes:
+                if Author.objects.filter(url=like.author.get_url_id()):
```

Comment:
Not sure what this if clause is doing? Doesn't it always return true?

Reply:
You don't have `exists()` and regardless I'm not sure that you need it

Reply:
You're right, it should actually have exist

Reply:
Actually, yeah, this shouldn't be here